collectionfunctions.php

```diff
@@ -36,16 +38,26 @@ function connectToDB(String $database): PDO {
     return $dbConnection;
 }
 
-/** Collects all pokemon values from pokemon database
+/** Collects all Pokémon values from Pokémon database
  * @param PDO $database
  * @return array
  */
 function collectData(PDO $database): Array {
-    $getPokemon = $database->prepare('SELECT `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats`;');
+    $getPokemon = $database->prepare("SELECT `stats`.`id`, `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats` WHERE `deleted` = 0;");
```

Comment:
If all the fields are in the stats table do you need to append `stats`. to each of them?

Reply:
Probably not 🧐 I think I put them in there as I thought it might be best practice in case it's swapped for a join but idk